### DISCUSSION TITLE
What happened to ContentView

### DISCUSSION THREAD

**Discussion by williambuchanan2** [dotnet/maui#6149] [General]
Can anyone tell me what happened to ContentView in Maui?

When you add one to a project it just creates a ResourceDictionary. Does anyone have examples of how I can use this in the same way as a ContentView?

Thanks

**Comment by DoubleDBE** [comment]
A little search would've yielded #6125 🤷‍♂️

**Reply by williambuchanan2** [reply]
Ugh, fair point. Thanks!
